cmd/kubeadm/app/phases/uploadconfig/uploadconfig.go

```diff
@@ -29,6 +30,14 @@ import (
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/util"
 	"k8s.io/kubernetes/cmd/kubeadm/app/util/apiclient"
+	rbachelper "k8s.io/kubernetes/pkg/apis/rbac/v1"
+)
+
+const (
+	// BootstrapDiscoveryClusterRoleName sets the name for the ClusterRole that allows
+	// the bootstrap tokens to access the kubeadm-config ConfigMap during the node bootstrap/discovery
+	// phase for additional master nodes
+	BootstrapDiscoveryClusterRoleName = "kubeadm:bootstrap-discovery-kubeadm-config"
```

Comment:
maybe in kubeadmconstants?

Reply:
This constant is used only here (e.g. similar to constants in pkg `k8s.io/kubernetes/cmd/kubeadm/app/phases/bootstraptoken`). No problem to move it